<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Toolbox</class>
 <widget class="QWidget" name="Toolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Toolbox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Demos</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="resetEverythingButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Reset!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sideBySideDemo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Side-by-side</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="randomiseModels">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Randomise Models</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>416</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Projection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioPerspective">
         <property name="text">
          <string>Perspective</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="perspectiveOptions">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="fov">
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
            <property name="value">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Aspect ratio</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="aspectRatio">
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="resetAspect">
            <property name="text">
             <string>Actual</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Field of View:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioOrtho">
         <property name="text">
          <string>Orthographic</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="grid">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Near Clip plane</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="nearClip">
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Far Clip plane</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="farClip">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>perspectiveOptions</zorder>
      <zorder>radioPerspective</zorder>
      <zorder>radioOrtho</zorder>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>316</width>
        <height>399</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Model</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="modelIndex"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="resetTransform">
         <property name="text">
          <string>Reset Transform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Translate</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="xTranslate">
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="yTranslate">
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Z:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="zTranslate">
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="translateButton">
            <property name="text">
             <string>Translate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Rotate</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Axis:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="rotationAxes">
            <item>
             <property name="text">
              <string>X-Axis</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Y-Axis</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Z-Axis</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Rotation:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="degreesRotation">
            <property name="suffix">
             <string>˚</string>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="rotateButton">
            <property name="text">
             <string>Rotate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Observer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="observerPerspective">
         <property name="text">
          <string>Perspective</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="observerPlanView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Plan View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="observerFrontView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Front View</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="observerSideView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Side View</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>391</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="matrixGroup">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Matrix</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
